.github/workflows/ci.yml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - "**.go"
+      - "go.mod"
+      - "go.sum"
```

Comment:
The only file that would not trigger this workflow is release.yml? Can we just ignore it?

Reply:
Habit. Will refine.